cpp/community_detection_module/CMakeLists.txt

```diff
@@ -13,15 +13,17 @@ target_include_directories(community_detection PRIVATE ${GRAPPOLO_HEADERS} ${CMA
 target_link_libraries(community_detection PRIVATE mg_utility basic_cd full_syn_opt coloring util OpenMP::OpenMP_CXX)
 
 # Module tests
-include(GoogleTest)
-set(community_detection_test_src
-    community_detection_test.cpp
-    algorithm/louvain.cpp)
+if (NOT MAGE_CUGRAPH_ENABLE)
```

Comment:
Why is here `if (NOT MAGE_CUGRAPH_ENABLE)`?
Does it mean we don't test with the `MAGE_CUGRAPH_ENABLE` flag ON?

Reply:
No tests are being built when cuGraph is enabled, because of the collision with GoogleTests, we need to figure out a better solution when we will have a dedicated machine